include/wx/combo.h

```diff
@@ -434,10 +434,10 @@ class WXDLLIMPEXP_CORE wxComboCtrlBase : public wxControl,
     }
 
     // These methods return references to appropriate dropbutton bitmaps
-    const wxBitmap& GetBitmapNormal() const { return m_bmpNormal; }
-    const wxBitmap& GetBitmapPressed() const { return m_bmpPressed; }
-    const wxBitmap& GetBitmapHover() const { return m_bmpHover; }
-    const wxBitmap& GetBitmapDisabled() const { return m_bmpDisabled; }
+    const wxBitmapBundle& GetBitmapNormal() const { return m_bmpNormal; }
+    const wxBitmapBundle& GetBitmapPressed() const { return m_bmpPressed; }
+    const wxBitmapBundle& GetBitmapHover() const { return m_bmpHover; }
+    const wxBitmapBundle& GetBitmapDisabled() const { return m_bmpDisabled; }
```

Comment:
Unlike the change in the other direction, this is not backwards compatible (`wxBitmap` is convertible to `wxBitmapBundle`, but not vice versa), so we can't do it like this. We need to preserve the existing functions unchanged and, if necessary, add accessors returning `wxBitmapBundle` (but IMO we shouldn't add them unless they're really needed).